cmd/nerdctl/main_test.go

```diff
@@ -88,3 +90,58 @@ version = 2
 	base.Env = append(base.Env, "NERDCTL_TOML="+tomlPath)
 	base.Cmd("info").AssertFail()
 }
+
+// TestNerdctlSpecificConfig validates the configuration precedence [CLI, TOML, Default].
+func TestNerdctlSpecificConfig(t *testing.T) {
+	testutil.DockerIncompatible(t)
+	t.Parallel()
+	tomlPath := filepath.Join(t.TempDir(), "nerdctl.toml")
+	err := os.WriteFile(tomlPath, []byte(`
```

Comment:
remove the file after the test complete

here's the demo:  https://github.com/containerd/nerdctl/pull/1150/files#diff-4843eb8080d6ff754054971c8db0ad20041e350810c18ba82302907eefcf77ecR475-R479

Reply:
fixed